lib/Translations/Download.php

```diff
@@ -40,8 +43,11 @@ public function download() : array {
 		$this->authenticate();
 		$downloads = [];
 
+		$lock          = LockHandler::get_instance();
+		$last_modified = $lock[ $this->config->get_name() ]['Last-Modified'] ?? '';
```

Comment:
Really not a fan of this. We have an object, why not use methods with signatures instead of a free for all array which is much more error prone?